src/components/button-toggle-group/button-toggle-group.stories.mdx

```diff
@@ -35,7 +35,6 @@ import ButtonToggleGroup from "carbon-react/lib/components/button-toggle-group";
   <Story name="default" parameters={{ info: { disable: true } }}>
```

Comment:
Maybe worth moving this to a `tsx` file so we can get the benefit of type checking, it's probably out of scope for this ticket so don't worry about it you'd prefer to leave it for now.

Reply:
I think @nineteen88 was addressing all of these in his PR that's currently in review. There may be some merge conflicts from that but it'll be worse if I attempt to do the same thing.

Reply:
Good point, happy for this to be addressed later if needs be 👍